.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: TerminalImageViewer CI
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "*" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Install dependencies
+      run: sudo apt-get install -qy imagemagick
+    - name: Build
+      run: make -C src
+    - name: Install
+      run: sudo make -C src install
+    - name: Test
+      run: tiv -w 80 -h 24 -2 /usr/share/pixmaps/debian-logo.png
```

Comment:
```suggestion
      run: tiv /usr/share/pixmaps/debian-logo.png
```
I don’t see why we should specify 8 bit color or the default size as the size, and this would make detection of terminal size untested. We should also compare this to an output that would be correct.

Reply:
The reasoning behind the hard-coded dimensions was to fix the warning it produces since there's no terminal associated with the process when run as a GitHub Action and it can't detect the terminal size:

    Warning: failed to determine most reasonable size, defaulting to 80x24
    
However, even with specifying that, it still prints the warning so I might look into that later. As for the the option to select 256-color mode, that was to try and improve the rendering in the log output as I don't know if whatever script GitHub is using to convert it to HTML was colorizing it correctly. Here is the original output before adding the options:

https://github.com/penguin359/TerminalImageViewer/actions/runs/7539833513/job/20523145272#step:6:6

And after adding them:

https://github.com/penguin359/TerminalImageViewer/actions/runs/7551617594/job/20559124690#step:6:6

That later one looks a lot better, im my opinion, but both do get a proper test of tiv so we can remove them if you prefer.

Reply:
Hmm, I was under the impression that GitHub output doesn’t display them at all.

The warning thing is a bug, along with the default actually being 20x6 instead of 80x24. I’ll try to fix these. Regardless, I don’t think color mode is the problem.

Reply:
Well, that might be a better explanation for why it looked so bad in the original test without the dimensions specified. I've removed the install and color palette restriction. Here's the latest run of that:

https://github.com/penguin359/TerminalImageViewer/actions/runs/7588182984/job/20670147766#step:5:6